scripts/generate-release-notes.py

```diff
@@ -105,6 +105,49 @@ def get_latest_repo_release_date(repo_org: str, repo_name: str) -> datetime:
     )
 
 
+def get_latest_and_previous_tags(
+    repo_org: str, repo_name: str, github_token: str
+) -> tuple:
+    """
+    Retrieves the latest and the previous release tags for the specified repository.
+    """
+    headers = {
+        "Authorization": f"Bearer {github_token}",
+        "Accept": "application/vnd.github.v3+json",
+    }
+    response = httpx.get(
+        f"https://api.github.com/repos/{repo_org}/{repo_name}/tags", headers=headers
+    )
```

Comment:
Probably not a big deal, but always nice to call `raise_for_status` so we get tracebacks if the call fails.

```suggestion
    )
    response.raise_for_status()
```

Reply:
sounds good!